src/store/counterSlice.ts

```diff
@@ -1,15 +1,15 @@
 import { createSlice, PayloadAction } from '@reduxjs/toolkit'
 
-interface ExampleState {
+interface CounterState {
   value: number
 }
 
-const initialState: ExampleState = {
+const initialState: CounterState = {
   value: 0,
 }
 
-const exampleSlice = createSlice({
-  name: 'example',
+const counterSlice = createSlice({
+  name: 'counter',
   initialState,
   reducers: {
     increment: (state) => {
```

Comment:
a minor comment for better: make constant for reducer name. It is just a personal taste 
exp: [EXAMPLE.INCREMENT]: ....

Reply:
RTK does not automatically create action creators for reducer keys written as constants

Reply:
DIY bro 
if cannot do manually, using AI for helping

Reply:
I asked AI for help and it recommended not to use constant for reducer name
you still want to DIY?